### ISSUE TITLE
Cop idea: Implicit Options Hash

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#12359]
## Description

Important: As of Ruby 2.7 braces around an options hash are no longer optional.

Omit the outer braces around an implicit options hash.

https://rubystyle.guide/#no-braces-opts-hash

## Describe the solution you'd like

```ruby
# bad
user.set({ name: 'John', age: 45, permissions: { read: true } })

# good
user.set(name: 'John', age: 45, permissions: { read: true })
```

**Comment by koic** [comment]
Whether braces can be omitted depends on the method definition, so it can't be detected uniformly.
cf: https://github.com/rubocop/rubocop/pull/7643
